src/components/characterList.jsx

```diff
@@ -18,7 +19,7 @@ const CharacterList = ({ scriptOption }) => {
       .catch((error) => {
         console.error(`error: ${error} when fetching character data`);
       });
-  }, []);
```

Comment:
In future PRs, a quick explanation about _why_ adding this variable here fixes the bug would be helpful